src/stylesheets/settings/_settings-components.scss

```diff
@@ -93,6 +93,21 @@ $theme-identifier-identity-domain-color: "base-light" !default;
 $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// Process List
```

Comment:
It seems like this could be a large amount of theme options, which we went over a little bit. The way I've approached some of what i've worked on is mainly theming through colors, and and looking at other theme settings that folks might want to adjust to keep things consistent. I don't quite know what the sweet spot is. Perhaps @thisisdano might have thoughts?

Reply:
Awesome feedback, I'll reduce the theme options to mostly color-related options

Reply:
In looking back over this, I kept it similar to Step Indicator. WIth Process List, there are pretty much the same customizations like color, font family, font size and then bullet size and connector size. Is this OK? @thisisdano ?